lib/index.js

```diff
@@ -190,23 +190,33 @@ Mixpanel.prototype.page = function(msg, fn){
   var category = msg.category();
   var name = msg.name();
   var fullName = msg.fullName(); // includes both category and name
+  var trackCategorizedPages = this.settings.trackCategorizedPages;
+  var trackNamedPages = this.settings.trackNamedPages;
+  var genericPageName = this.settings.genericPageName;
+  var trackAllPages = this.settings.trackAllPages;
   var self = this;
 
   // all
-  if (this.settings.trackAllPages) {
+  if (trackAllPages){
     return send(msg.track(), fn);
   }
 
+  // if they want to send all page/scren events with the default name from facade
+  // i.e. "Loaded a Screen", "Loaded a Page"
+  if (genericPageName){
+    //  but still respect their other settings
+    if (trackCategorizedPages && category) return send(msg.track(), fn);
```

Comment:
I wonder if this is a good time to get rid of the category/name setting? Leave it for existing users, but if you've opted into generic page name (or whatever we call it), I think this shouldn't be a choice anymore.